lib/jsonapi/serializer.ex

```diff
@@ -144,7 +189,9 @@ defmodule JSONAPI.Serializer do
 
   defp include_view(view, _key), do: generate_view_tuple(view)
 
+  defp generate_view_tuple({_rewrite_key, view, :include}), do: {view, :include}
   defp generate_view_tuple({view, :include}), do: {view, :include}
+  defp generate_view_tuple({_rewrite_key, view}), do: {view, :include}
```

Comment:
Does this result in us always including a view if it's relationship name is overwritten?